src/awsme/dummy.py

```diff
@@ -2,3 +2,7 @@ class DummyCloudWatch:
 
     def log(self, *args, **kwargs) -> None:
         pass
+
+    def flush(self, complete: bool = True) -> None:
+        pass
+        
```

Comment:
Please get rid of the "No newline at end of file", I hate them.

Reply:
Added the newline - I think. Please check, as diff still seems to think its not there, at least how I'm reading it

Reply:
Yep, it's good now.